File: BackEnd/src/main/java/com/BackEnd/dto/OrderDTO.java
package com.BackEnd.dto;

import com.BackEnd.model.Order;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;


@Data
@AllArgsConstructor
@NoArgsConstructor
public class OrderDTO {
    private Long orderId;
    private Long orderCode;
    private String userName;
    private LocalDateTime createTime;
    private Double totalPrice;
    private Order.OrderStatus status;
    private String qrCodeToCheckout;

}

File: FrontEnd/app/src/main/java/com/example/frontend/data/dto/OrderDTO.kt
package com.example.frontend.data.dto

import java.time.LocalDateTime

data class OrderDTO(
    val orderId: Long,
    val orderCode: Long,
    val userName: String,
    val createTime: String,
    val totalPrice: Double,
    val status: OrderStatus,
    val qrCodeToCheckout: String,
    val orderDetailDTOList: List<OrderDetailDTO>,
)
